project-generator/src/main/java/io/syndesis/project/converter/DefaultProjectGenerator.java

```diff
@@ -104,9 +123,31 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
         });
 
         Map<String, byte[]> contents = new HashMap<>();
+
+        for (Templates.Resource additionalResource : generatorProperties.getTemplates().getAdditionalResources()) {
+            String overridePath = generatorProperties.getTemplates().getOverridePath();
+            URL resource = null;
+
+            if (!Strings.isEmpty(overridePath)) {
+                resource = getClass().getResource("templates/" + overridePath + "/" + additionalResource.getSource());
```

Comment:
What if overridePath is provided but the resource could not be found ? The overridePath should be at least included in the error message, but maybe already throw an exception here.